websites/B/Battlefield Portal/dist/metadata.json

```
@@ -0,0 +1,49 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.5",
+    "service": "Battlefield Portal",
+    "author": {
+      "name": "gala",
+      "id": "338947895665360898"
+    },
+    "category": "games",
+    "logo": "https://i.imgur.com/kEnCDvm.png",
+    "thumbnail": "https://i.imgur.com/3WooHwa.png",
+    "version": "1.0.0",
+    "color": "#26ffe4",
+    "tags": [
+      "battlefield2042",
+      "portal",
+      "rules-editor"
+    ],
+    "url": "portal.battlefield.com",
+    "description": {
+      "en": "Battlefield Portal is a platform where you will discover unexpected battles and change the rules of war. Take on custom-made modes or create your own unique experience using powerful builder tools."
+    },
+    "settings": [
+      {
+        "id": "show_block",
+        "title": "Show which block/subroutine your editing",
+        "value": true
+      },
+      {
+        "id": "elapsed_time",
+        "title": "Show Elapsed Time",
+        "value": true
+      },
+      {
+        "id": "button",
+        "title": "Show button to open experience",
```

```suggestion
        "title": "Show button",
```

---

Should probably be plural?
```suggestion
        "title": "Show buttons",
```

---

There will be only one button

---

Buttons is still more correct and future-proof, regardless of how many there are